autometa/taxonomy/gtdb.py

```diff
@@ -76,17 +78,20 @@ def create_gtdb_db(reps_faa: str, dbdir: str) -> str:
         os.makedirs(dbdir)
 
     logger.debug(f"Merging {len(faa_index):,} faa files.")
-    combined_faa = os.path.join(dbdir, "gtdb.faa")
+    combined_faa = os.path.join(dbdir, "gtdb.faa") 
     with open(combined_faa, "w") as f_out:
         for faa_file, acc in faa_index.items():
-            with open(faa_file) as f_in:
+            with gzip.open(faa_file, "rb") as f_in:
                 for line in f_in:
-                    if line.startswith(">"):
-                        seqheader = line.lstrip(">")
+                    # print(line.decode('utf-8'))
+                    if line.decode('utf-8').startswith(">"):
+                        seqheader = line.decode('utf-8').lstrip(">")
                         line = f"\n>{acc} {seqheader}"
```

Comment:
- :fire: print statement
- :art: Can remove redundancy and times `str.decode('utf-8')` is called (DRY) with a single line
- :art: Perhaps a different variable for the line being read vs. written could improve readability?

> Some ternary operators may allow handling both compressed and decompressed files here

```suggestion
                    line = line.decode("utf-8")
                    if line.startswith(">"):
                        seqheader = line.lstrip(">")
                        outline = f"\n>{acc} {seqheader}"
```